tests/conftest.py

```diff
@@ -253,3 +254,17 @@ def any_changelog_format(config: BaseConfig) -> ChangelogFormat:
     """For test not relying on formats specifics, use the default"""
     config.settings["changelog_format"] = defaults.CHANGELOG_FORMAT
     return get_changelog_format(config)
+
+
+@pytest.fixture(autouse=True)
+def default_change_type_map(mocker: MockerFixture) -> None:
+    mocker.patch.object(
+        ConventionalCommitsCz,
+        "change_type_map",
+        {
+            "feat": "Feat",
```

Comment:
Why are the tests using the old format?

Reply:
I did not want to change the tests. So I applied mocking instead to hard-coding the new values.

IMHO: For tests, the names should not matter, I guess. The point is what the values are actually used for.

Reply:
It might be a bit confusing as these are no longer the actual `default`_change_type_map when someone's tracing the test cases, they might be misled